alignak/alignakobject.py

```diff
@@ -20,70 +20,159 @@
 #
 """ This module contains only a common class for all object created in Alignak: AlignakObject.
 """
-
+from __future__ import print_function
```

Comment:
You should have tried to debug this with logger.debug and yhou will understand why I used print :/